src/components/WarningAnimateMessage/index.js

```diff
@@ -99,8 +99,8 @@ const styles = StyleSheet.create({
     elevation: 1,
     borderRadius: 8,
     width: 259,
-    backgroundColor: COLORS.white,
-    borderColor: COLORS.concrete,
+    backgroundColor: COLORS.almostBlack,
+    borderColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `backgroundColor` and `borderColor` properties seem fine. However, it's important to ensure that these color changes are consistent with the overall theme of the application and provide a good user experience in terms of readability and contrast.

```diff
-    backgroundColor: COLORS.white,
-    borderColor: COLORS.concrete,
+    backgroundColor: COLORS.almostBlack,
+    borderColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->